ros/src/computing/planning/common/lib/astar_search/src/astar_util.cpp

```diff
@@ -28,10 +28,8 @@
  *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-#include "astar_util.h"
+#include "astar_search/astar_util.h"
 
-namespace astar_planner
-{
 WaveFrontNode::WaveFrontNode()
```

Comment:
Struct member variables should be initialized also by the empty constructor to ensure non-random initial variable values. The same should happen with _SimpleNode_ below

```suggestion
WaveFrontNode::WaveFrontNode() : index_x(0), index_y(0), hc(0)
```